classifier/FPhi.py

```diff
@@ -1,39 +1,42 @@
 from torch import nn
-from pytorch_lightning import Trainer
 import torch.utils.data
-from torch.utils.data import DataLoader
 import torch.optim
 
 # Pytorch Lightning
 import pytorch_lightning as pl
 from pytorch_lightning import Trainer
-from pytorch_lightning.callbacks import ModelCheckpoint, EarlyStopping
-from pytorch_lightning.logging import TestTubeLogger
 
 import os, sys
 from argparse import ArgumentParser
-import datetime
 
 
 class FPhi(pl.LightningModule):
-    def __init__(self, hparams,sizeOfInputNode):
+    def __init__(self, hparams):
         super(FPhi, self).__init__()
         self.hparams = hparams
 
-        nb_nodes =  sizeOfInputNode#Input size is #ofJoints**2
+        if hparams.dataset == "PHYT":
+            final_layer_count = 3
+        elif hparams.dataset == "SBU":
+            final_layer_count = 8
+
+        if hparams.intra:
+            first_layer_size = 1000
+        elif hparams.inter:
+            first_layer_size = 500
+
```

Comment:
Does SBU have inter and Intra?

Reply:
yeah, it's also a skeleton but with 15 joints